content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md

```diff
@@ -14,21 +14,72 @@ CatalogContent:
 
 **Multi-factor Authentication (MFA)** is the use of more than one factor to prove the identity of a user.
 
+One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require users to create when initially signing up for an account.
```

Comment:
Can we create a segue to the other factors below

Reply:
I am not sure I understand this request. Do you think it pertinent to add another sentence after line 17?

Reply:
After a second look, I think this is straightforward and good as-is 🙂